python/Ganga/Utility/execute.py

```diff
@@ -283,7 +283,7 @@ def execute(command,
         if stdout:
             stdout = pickle.loads(stdout)
     except Exception as err:
-        logger.error("Err: %s" % str(err))
+        logger.debug("Err: %s", err)
```

Comment:
This is just hiding problems isn't it? We normally don't get here anyway as most uses don't use the stdout I think